ui/src/tasks/actions/index.ts

```diff
@@ -296,7 +321,7 @@ export const updateTaskStatus = (task: Task) => async dispatch => {
 
 export const updateTaskName = (task: Task) => async dispatch => {
```

Comment:
Right, I totally agree! Especially since we are literally doing just that where it's being passed in:

https://github.com/influxdata/influxdb/blob/master/ui/src/tasks/components/TaskCard.tsx#L155-L158

Should I refactor it?